mrtt-ui/src/App.js

```diff
@@ -27,11 +27,13 @@ function App() {
             <Route path='/' element={<Home />} />
             <Route path='/landscapes' element={<Landscapes />} />
             <Route path='/organizations' element={<Organizations />} />
-            <Route path='/site/:siteId' element={<Site />} />
+            <Route path='/site/:siteId/edit' element={<SiteForm isNewSite={false} />} />
             <Route path='/site/:siteId/form/causes-of-decline' element={<CausesOfDeclineForm />} />
             <Route path='/site/:siteId/form/project-details' element={<ProjectDetailsForm />} />
             <Route path='/site/:siteId/form/restoration-aims' element={<RestorationAimsForm />} />
             <Route path='/site/:siteId/form/site-background' element={<SiteBackgroundForm />} />
+            <Route path='/site/:siteId/overview' element={<SiteOverview />} />
+            <Route path='/site/new' element={<SiteForm isNewSite={true} />} />
```

Comment:
I'm noticing that you're explicitly stating boolean props. 

I'm used to the pattern of either <FancyThing /> or omission. Why are you choosing this path?